packages/cli/src/lib/commands/space.ts

```diff
@@ -148,17 +148,33 @@ export const space: CommandDefinition = (program) => {
 
     accessKeyCmd.command("revoke")
         .description("Revokes Access Key in active Space")
-        .argument("<access_key>", "Access Key id")
-        .action(async (accessKey: string) => {
+        .option("--id <id>", "Hub Id")
+        .option("--all", "Disconnects all self-hosted Hubs connected to Space")
+        .action(async (options: { all: boolean, id: string }) => {
             const spaceName = sessionConfig.lastSpaceId;
             const mwClient = getMiddlewareClient();
 
             if (!spaceName) {
                 throw new Error("No Space set");
             }
+            if (options.all) {
+                const apiKeys = await mwClient.listAccessKeys(spaceName);
+
+                if (!apiKeys.accessKeys || apiKeys.accessKeys.length === 0) {
+                    throw new Error("There are no keys to revoke");
+                } else {
+                    const revokedAccessKeys = await mwClient.revokeAllAccessKeys(spaceName, apiKeys);
 
-            const revokedAccessKey = await mwClient.revokeAccessKey(spaceName, accessKey);
+                    displayObject(revokedAccessKeys, profileManager.getProfileConfig().format);
 
-            displayObject(revokedAccessKey, profileManager.getProfileConfig().format);
+                    process.exit(1);
+                }
+            } else if (options.id) {
+                const revokedAccessKey = await mwClient.revokeAccessKey(spaceName, options.id);
+
+                displayObject(revokedAccessKey, profileManager.getProfileConfig().format);
+            } else {
```

Comment:
Instead of nesting last, use negation of xor on top (This way you make sure user provides only one option specificaly):
```
if (!(all ^ Boolean(id)))
throw new Error("Please provide one of the options, please use command with --help to get more information");

```
please use command with --help to get more information - Please check, I'm not sure if it is not shown as default on error respones in CLI

Reply:
![image](https://github.com/scramjetorg/transform-hub/assets/53794300/778443b2-ef5f-4269-87f2-a2b2231c79d1)
Not shown in this scenario